src/sql/platform/query/common/queryManagement.ts

```diff
@@ -87,6 +89,7 @@ export class QueryManagementService implements IQueryManagementService {
 	public _serviceBrand: any;
 
 	private _requestHandlers = new Map<string, IQueryRequestHandler>();
+	private _onQueryRequestHandlerAddedEmitter = new Emitter<string>();
```

Comment:
Can this just be `onHandlerAddedEmitter`? We know it's a query request handler...